`make clean` fails

It appears that `make clean` fails when run on the Warehouse repository on my machine.  

Here is the error message I got: 
```
$ make clean
rm -rf warehouse/static/components
rm -rf warehouse/static/dist
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.a7bbeb58.woff': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.1fb8a705.eot.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.e1dd9adb.ttf.br': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.1fb8a705.eot.br': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/FontAwesome.b5744b64.otf.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/FontAwesome.b5744b64.otf': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.912ec66d.svg.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.e1dd9adb.ttf.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.912ec66d.svg': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.68560c51.woff2': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.a7bbeb58.woff.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/FontAwesome.b5744b64.otf.br': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.e1dd9adb.ttf': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.1fb8a705.eot': Permission denied
rm: cannot remove 'warehouse/static/dist/fonts/fontawesome-webfont.912ec66d.svg.br': Permission denied
rm: cannot remove 'warehouse/static/dist/manifest.json.br': Permission denied
rm: cannot remove 'warehouse/static/dist/images/favicon.6a76275d.ico': Permission denied
rm: cannot remove 'warehouse/static/dist/images/blue-cube.e6165d35.svg': Permission denied
rm: cannot remove 'warehouse/static/dist/images/logo-small.972290d5.svg.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/images/favicon.6a76275d.ico.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/images/logo-small.972290d5.svg': Permission denied
rm: cannot remove 'warehouse/static/dist/images/white-cube.2bc660f6.png': Permission denied
rm: cannot remove 'warehouse/static/dist/images/white-cube-small.cb1d84fb.png': Permission denied
rm: cannot remove 'warehouse/static/dist/images/logo-large.87e8a3ef.svg.br': Permission denied
rm: cannot remove 'warehouse/static/dist/images/blue-cube.e6165d35.svg.br': Permission denied
rm: cannot remove 'warehouse/static/dist/images/blue-cube-small.72d3cc02.png': Permission denied
rm: cannot remove 'warehouse/static/dist/images/white-cube.8c3a6fe9.svg.br': Permission denied
rm: cannot remove 'warehouse/static/dist/images/favicon.6a76275d.ico.br': Permission denied
rm: cannot remove 'warehouse/static/dist/images/logo-large.87e8a3ef.svg': Permission denied
rm: cannot remove 'warehouse/static/dist/images/blue-cube.e6165d35.svg.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/images/history-line.ded09064.png': Permission denied
rm: cannot remove 'warehouse/static/dist/images/white-cube.8c3a6fe9.svg.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/images/logo-small.972290d5.svg.br': Permission denied
rm: cannot remove 'warehouse/static/dist/images/logo-large.87e8a3ef.svg.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/images/white-cube.8c3a6fe9.svg': Permission denied
rm: cannot remove 'warehouse/static/dist/css/font-awesome.css.4741d860.map.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/css/warehouse.css.b3ca61fa.map.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/css/warehouse.css.b3ca61fa.map.br': Permission denied
rm: cannot remove 'warehouse/static/dist/css/warehouse.8aff0405.css.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/css/font-awesome.css.4741d860.map.br': Permission denied
rm: cannot remove 'warehouse/static/dist/css/font-awesome.628aab96.css.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/css/font-awesome.css.4741d860.map': Permission denied
rm: cannot remove 'warehouse/static/dist/css/warehouse.css.b3ca61fa.map': Permission denied
rm: cannot remove 'warehouse/static/dist/css/font-awesome.628aab96.css.br': Permission denied
rm: cannot remove 'warehouse/static/dist/css/warehouse.8aff0405.css': Permission denied
rm: cannot remove 'warehouse/static/dist/css/warehouse.8aff0405.css.br': Permission denied
rm: cannot remove 'warehouse/static/dist/css/font-awesome.628aab96.css': Permission denied
rm: cannot remove 'warehouse/static/dist/manifest.json': Permission denied
rm: cannot remove 'warehouse/static/dist/manifest.json.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/js/warehouse.80c5a887.js': Permission denied
rm: cannot remove 'warehouse/static/dist/js/warehouse.80c5a887.js.br': Permission denied
rm: cannot remove 'warehouse/static/dist/js/warehouse.js.2932ed70.map': Permission denied
rm: cannot remove 'warehouse/static/dist/js/warehouse.js.2932ed70.map.br': Permission denied
rm: cannot remove 'warehouse/static/dist/js/vendor/zxcvbn.js.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/js/vendor/zxcvbn.js': Permission denied
rm: cannot remove 'warehouse/static/dist/js/warehouse.80c5a887.js.gz': Permission denied
rm: cannot remove 'warehouse/static/dist/js/warehouse.js.2932ed70.map.gz': Permission denied
Makefile:156: recipe for target 'clean' failed
make: *** [clean] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -161,8 +161,6 @@ shell:
 	docker-compose run --rm web python -m warehouse shell
 
 clean:
-	rm -rf warehouse/static/components
-	rm -rf warehouse/static/dist
 	rm -rf dev/*.sql
 
 purge: stop clean

